lib/Controller/Widget.php

```diff
@@ -1638,6 +1666,10 @@ public function widgetSetRegion(Request $request, Response $response, $id)
             throw new InvalidArgumentException(__('This Layout is not a Draft, please checkout.'), 'layoutId');
         }
 
+        if ($playlist->isDynamic === 1) {
```

Comment:
Dynamic playlists are never region specific (they are always created on the playlist page, not the layout editor), so I think this can't be triggered)